src/rinterface_extra.c

```diff
@@ -3258,7 +3258,7 @@ int R_SEXP_to_sparsemat(SEXP pakl, igraph_sparsemat_t *akl) {
 
   igraph_sparsemat_view(akl, /*nzmax=*/ GET_LENGTH(x),
                           /*m=*/ INTEGER(Dim)[0], /*n=*/ INTEGER(Dim)[1],
-                          /*p=*/ INTEGER(p), /*i=*/ INTEGER(i),
+                          /*p=*/ (igraph_integer_t*)INTEGER(p), /*i=*/ (igraph_integer_t*)INTEGER(i),
```

Comment:
Please don't do this. This just hides the problem, it does not fix it.

This function is not in use at the moment, so there are no symptoms. If it were in use, it would of course crash if you do this.

Reply:
Okay, I will leave it for separate pull request which will fix sparse.